stories/Link.stories.ts

```diff
@@ -0,0 +1,64 @@
+import { Meta, Story } from '@storybook/angular';
+import { CommonModule } from '@angular/common';
+
+import { IonLinkComponent } from '../projects/ion/src/lib/link/link.component';
+import { IonLinkModule } from '../projects/ion/src/public-api';
+
+export default {
+  title: 'Ion/Navigation/Link',
+  component: IonLinkComponent,
+} as Meta;
+
+const Template: Story<IonLinkComponent> = (args: IonLinkComponent) => ({
+  component: IonLinkComponent,
+  props: { ...args },
+  moduleMetadata: {
+    imports: [CommonModule, IonLinkModule],
+    entryComponents: [IonLinkComponent],
+  },
+});
+
+export const Default = Template.bind({});
+Default.args = {
+  label: 'Link',
+  link: 'https://github.com/',
+};
+
+export const WithIcon = Template.bind({});
```

Comment:
Would it be OK to do an example with the icon aligned to the left?

Reply:
The dev can change the side with the controls, but i can add this case